easybuild/easyconfigs/p/PEXSI/PEXSI-012276f-intel-2020b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'PEXSI'
+version = '012276f'
```

Comment:
commit hashes are not very useful as versions, please consider setting `version` to be e.g. the date of the commit, in `YYYYMMDD` format, and define e.g. `local_commit = '012276f'` to use in the `git_config` below